check_http.go

```diff
@@ -41,20 +43,22 @@ func (c CheckHTTP) Execute(timeout time.Duration) error {
 
 	req, err := http.NewRequest(c.Method, c.Path, nil)
 	if err != nil {
-		return err
+		return false, fmt.Errorf("building http request failed %s", err)
 	}
 
 	res, err := client.Do(req)
 	if err != nil {
-		return err
+		log.Println(log.LevelInfo, "HTTP request failed: %s", err)
```

Comment:
Is there a reason to log these directly? I don't like logging from non-main packages as it cannot be configured/overloaded (like pythons `logging`). Could we turn these into returned errors?